.gitlab-ci.yml

```diff
@@ -68,10 +68,14 @@ test:extra-tools:release-tool:
     - git fetch github
     # Fetch master branch for tests using --in-integration-version
     - git fetch origin master:master
+    - git fetch origin --tags
+    - git submodule update --init --recursive
 
   script:
     # Run release-tool unit tests.
     - python3 -m pytest extra/test_release_tool.py
+    - git config --global mailmap.file $(pwd)/extra/gitdm/mailmap
+    - python3 -m pytest extra/test_statistics_generator.py
```

Comment:
There is a different job defined above which tests the changelog generator; I think it fits better there.

But on the other hand, I think it's overkill to have separate jobs for these tests, they execute in fractions of seconds. Maybe you can just squash this job with the previous job and call it `test:extra-tools`?

Reply:
Sure, @lluiscampos unless you object

Reply:
Makes sense to group them :+1: